astropy/tests/runner.py

```diff
@@ -126,6 +138,16 @@ def __new__(cls, *args, **kwargs):
                 doc_keywords += func.__doc__.strip()
                 doc_keywords += '\n\n'
 
+            if 'plugins' not in cls.keywords or cls.keywords['plugins'] is None:
+                cls.keywords['plugins'] = []
+
+            # Make plugins available to test runner without registering them
+            cls.keywords['plugins'].extend([
+                'astropy.tests.plugins.display',
+                'astropy.tests.plugins.repeat',
+                'astropy.tests.plugins.config'
+            ])
+
```

Comment:
@astrofrog see here for how we make the plugins available to `astropy.test()` without registering them through `setup.py`.